main.js

```diff
@@ -0,0 +1,88 @@
+var board = [
+  [5, 4, 3, 2, 1],
+  [],
+  [],
+];
+
+
+var gameBoard = {
+  board: [
+    [5, 4, 3, 2, 1],
+    [],
+    [],
+  ]
+}
+
+//the state of the board that is printed after each move
+var printBoard = function() {
+  board.map(function(disc) {
+    var discAppearance = disc.join(" ");
+    console.log("---" + discAppearance);
+})};
+
+
+//checks to see if either peg 2 or 3 has all 5 discs on it
+var checkWinner = function() {
+  if (board[1].length === 5 || board[2].length === 5) {
+    console.log('Congrats, you win! Play again?');
+    board = [
+      [5, 4, 3, 2, 1],
+      [],
+      [],
+    ];
+    printBoard();
+  }
+};
+
+var lastDiscOnPeg = function(peg) {
+  return board[peg].at(-1);
+};
+
+// checks to ensure there are valid moves
+var checkMove = function(currentPeg, targetPeg) {
```

Comment:
Would recommend adding some helper functions to reduce how much is in this function